.enb/helpers/page.js

```diff
@@ -0,0 +1,171 @@
+ *     project.node('desktop.bundles/index', function (node) {
+ *         // provide BEMJSON file
+ *         node.addTech([FileProviderTech, { target: '?.bemjson.js' }]);
+ *
+ *         // build page
+ *         configurePage(node, {
+ *             platform: 'desktop',
+ *             langs: ['en', 'ru']
+ *         });
+ *     });
+ * };
+ */
+module.exports = function(node, opts) {
+    var nodeUrl = node.getPath();
+
+    opts = opts || {};
+
+    if(!opts.platform) {
+        var error = new Error('platform to configure the `' + nodeUrl + '` node is not specified.');
+        error.name = 'ConfigError';
+
+        throw error;
+    }
+
+    var langs = opts.langs || [],
+        nodeDirname = node.getNodePath(),
+        sublevels = [
+            path.join(nodeDirname, '..', '.blocks'),
+            path.join(nodeDirname, 'blocks')
+        ].filter(function (level) {
+            return fs.existsSync(level);
+        }),
+        levels = [].concat(
+            config.levels(opts.platform),
+            sublevels
+        ),
+        needBEMTREE = opts.bemtree;
+
+    node.addTechs([
+        // get FileList
+        [techs.bem.levels, { levels : levels }],
+        !needBEMTREE && [techs.bem.bemjsonToBemdecl],
+        [techs.bem.depsOld],
+        [techs.bem.files],
+
+        // build CSS
+        [techs.css],
+
+        // build BEMHTML for browsers
+        [techs.bem.depsByTechToBemdecl, {
+            target : '?.bemhtml.bemdecl.js',
+            sourceTech : 'js',
+            destTech : 'bemhtml'
+        }],
+        [techs.bem.depsOld, {
+            target : '?.bemhtml.deps.js',
+            sourceDepsFile : '?.bemhtml.bemdecl.js'
+        }],
+        [techs.bem.files, {
+            target : '?.bemhtml.deps.js',
+            filesTarget : '?.bemhtml.files',
+            dirsTarget : '?.bemhtml.dirs'
+        }],
+        [techs.engines.bemhtml, {
+            target : '?.browser.bemhtml.js',
+            filesTarget : '?.bemhtml.files',
+            devMode : false
+        }],
+
+        // build JavaScript for browsers
+        [techs.js, {
+            target : '?.browser.js',
+            sourceSuffixes : ['vanilla.js', 'js', 'browser.js']
+        }],
+        [techs.files.merge, {
+            target : '?.pre.js',
+            sources : ['?.browser.bemhtml.js', '?.browser.js']
+        }],
+        [techs.ym, {
+            source : '?.pre.js',
+            target : '?.js'
+        }],
+    ].filter(function (tech) {
+        return tech;
+    }));
+
+    if(needBEMTREE) {
+        // build HTML using BEMTREE
+        node.addTechs([
+            [techs.engines.bemhtml],
+            [techs.engines.bemtree],
+            [techs.html.bemtree]
+        ]);
+    } else {
+        node.addTechs([
+            // build HTML using BEMHTML
+            [techs.engines.bemhtml],
+            [techs.html.bemhtml],
+
+            // build HTML using BH
+            [techs.engines.bhCommonJS, {
```

Comment:
No, but this PR about refactoring only.

All improvements suggest doing separately.